infusoria.py

```diff
@@ -191,12 +191,11 @@ def sex(u):
             mutation_count = mutation_count + 1
             #pygame.time.delay(1000)
             if debug:
-                print >> sys.stderr, '############### MUTATION ################ count:', mutation_count
-
- 	if debug:
-            print >> sys.stderr, '!!!!!!!!!!!!!!!!!!!!!!CLONING!!!!!!!!!!!!!!!!!!!!!!!!!!', len(unit), 'units', '       sex_count', sex_count
-            for i in xrange(len(unit)):
-                print >> sys.stderr, i+1, '--', 'age:', unit[i]['AGE'], '   eating:', unit[i]['EAT_COUNT'], '  hunger:', unit[i]['HUNGRY'], '   color:', unit[i]['COLOR']
+                print('############### MUTATION ################ count:', mutation_count)
```

Comment:
Here i want to see any different facility of output. In  my version i use `>> sys.stderr` consttruction.
But it not critical.